tarok/test/cards_tests.cpp

```diff
@@ -53,9 +54,31 @@ TEST_F(CardsTests, TestCountCards) {
   std::iota(all_card_actions.begin(), all_card_actions.end(), 0);
   EXPECT_EQ(tarok::CardPoints(all_card_actions, deck), 70);
```

Comment:
This is my mistake, may I please ask you to remove all `tarok::` here as we are in the `tarok` namespace?

Reply:
Done.